server/src/main/java/io/crate/metadata/pgcatalog/PgAttributeTable.java

```diff
@@ -73,4 +76,16 @@ public static SystemTable<ColumnContext> create() {
             .add("attmissingval", STRING_ARRAY, c -> null)
             .build();
     }
+
+    /**
+     * For dropped columns we show only top-level column name
+     * to display prefix and not something like o['_dropped_5']
+     */
```

Comment:
How is the top-level situation related to this method? I'd suggest to remove the comment, it's only mapping related (we drop all childs) and not how things are exposed.